admin/admin-menu-and-tabs.php

```diff
@@ -67,8 +67,17 @@ public function register_menu() {
     }
 
     public function enqueue_scripts() {
-        wp_enqueue_script( 'dt_campaign_admin_script', plugin_dir_url( __FILE__ ) . 'admin.js', [ 'jquery' ], filemtime( __DIR__ . '/admin.js' ), true );
+        wp_enqueue_script( 'bootstrap-js', 'https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.5.3/js/bootstrap.min.js', [ 'jquery' ], '4.5.3', true );
```

Comment:
@kodinkat do we need bootstrab. Does the normal jquery modal work?
Including bootstrap messes with the visual in general.

Reply:
Replacing boostrap with foundation results in the same issue.
Does the normal jquery modal like we use in the translations work?